docs/go/how-to-replay-a-workflow-execution-in-go.md

```diff
@@ -0,0 +1,61 @@
+---
+id: how-to-replay-a-workflow-execution-in-go
+title: How to replay a Workflow Execution in Go
+sidebar_label: Workflow replay
+description: Use the WorflowReplayer API to replay an existing Workflow Execution from an Event History to replicate errors.
+tags:
+  - go
+  - how-to
+---
+
+Use the [worker.WorflowReplayer](https://pkg.go.dev/go.temporal.io/sdk/worker#WorkflowReplayer) to replay an existing Workflow Execution from an Event History to replicate errors.
```

Comment:
This completely leaves off that history can be fetched via the web UI. Also we should probably add that `tctl` can obtain the history too. This is just an example of obtaining it programmatically if you really wanted to.